app/resources/documentation/manual/index.md

```diff
@@ -1242,17 +1242,18 @@ Expression Value
 A map is a comma-separated list of of key-value pairs, enclosed in braces. Note that keys are strings, and so must be quoted. The value is separated from the key by a colon character.
```

Comment:
"Note that keys are strings, and __so must be quoted.__"  No longer true as of this commit.